test/system/070-build.bats

```diff
@@ -393,9 +393,9 @@ Labels.$label_name | $label_value
        "image tree: third line"
     is "${lines[3]}" "Image Layers" \
        "image tree: fourth line"
-    is "${lines[4]}"  "...  ID: [0-9a-f]\{12\} Size: .* Top Layer of: \[$IMAGE]" \
+    is "${lines[4]}"  ".* ID: [0-9a-f]\{12\} Size: .* Top Layer of: \[localhost/build_test:latest]" \
        "image tree: first layer line"
-    is "${lines[-1]}" "...  ID: [0-9a-f]\{12\} Size: .* Top Layer of: \[localhost/build_test:latest]" \
+    is "${lines[-1]}" ".* ID: [0-9a-f]\{12\} Size: .* Top Layer of: \[$IMAGE]" \
        "image tree: last layer line"
```

Comment:
Just a comment, not a change request: this seems weird, like a deliberate reordering/reversal of image tree. Isn't this a breaking change? Will users be affected?

Reply:
Sorry for not mentioning that in the commit message here.  I only mentioned it in the last libimage commit (https://github.com/containers/common/commit/c53283f5fc9a7bdbf727478bc5e6e32ce490524a).

It was a deliberate reordering.  I had to rewrite image tree to make it use the internal layer tree (used to compute parent-child relationships).  When looking closer at it, the previous order did not feel intuitive as the top layer was at the very bottom.  In practice, however, layers are put on top of each other.  It also came with the nice benefit of making the code much simpler.

I see the output of `image tree` as something like the pull/push progress bars which may change over time (rather cosmetic) but others may feel different.

I will make it a PS in today's meeting.

Reply:
I did some more detailed testing of `image tree` and found some other issues in `--whatrequires`.  I will address them in a follow-up PR.